proto/iotics/api/common.proto

```diff
@@ -36,7 +33,7 @@ message Offset {
 }
 
 // Range is the combination of the "Limit" and "Offset" is a request parameters. It is
-// used to return a specific range of results.
+// used to return a specific range of results. Default server values are applied.
```

Comment:
Maybe:
> Default server values are applied if no limit is specified.

Reply:
Also, as above - should we say server or host or space?

Reply:
https://github.com/Iotic-Labs/api/pull/10/commits/bc0f8fbc37582bd2a44de7b9caa16848389b4cda 
Server or Host or Space => the 1 million pound question :grin: 
I removed the mention of server